src/parser/pegen.ts

```diff
@@ -1391,7 +1392,7 @@ export function singleton_seq(p: Parser, a: AST): AST[] {
 
 /* Flattens an asdl_seq* of asdl_seq*s */
 export function seq_flatten(p: Parser, seqs: AST[][]): AST[] {
-    return seqs?.flat();
+    return seqs?.flat(Infinity) as AST[];
```

Comment:
I'm not actually sure we need a dept of Infinity - Maybe we should leave as a depth of 1 and add this in if we find we need it?

Reply:
be7d1bd